conda_env/specs/__init__.py

```diff
@@ -2,13 +2,15 @@
 from .yaml_file import YamlFileSpec
 from .notebook import NotebookSpec
 from .requirements import RequirementsSpec
+from .jinja2_file import Jinja2Spec
 from ..exceptions import SpecNotFound
 
 all_specs = [
     BinstarSpec,
     NotebookSpec,
     YamlFileSpec,
-    RequirementsSpec
+    RequirementsSpec,
+    Jinja2Spec
```

Comment:
Actually, perhaps it is possible for a file to have jinja2 delimiters and still be a valid yaml file! In this case, your concern is still valid.